### PR TITLE
Doc: Include the support for multiple monitors starting CloudStack v4.18.0.0

### PR DESCRIPTION
**Pull request by zap51**
Doc: Include the support for multiple monitors starting CloudStack v4.18.0.0
Signed-off-by: Jayanth Reddy <45934027+zap51@users.noreply.github.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by zap51 — COMMENTED** [review]
Fixed typo in the PR.

File: doc/rbd/rbd-cloudstack.rst [hunk 1]
```diff
@@ -143,7 +143,7 @@ created (e.g., ``cloudstack``).
 Limitations
 ===========
 
-- CloudStack will only bind to one monitor (You can however create a Round Robin DNS record over multiple monitors)
+- CloudStack, until v4.17.1.0 will only bind to one monitor (You can however create a Round Robin DNS record over multiple monitors).
```
Comment (anthonyeleven) [inline doc/rbd/rbd-cloudstack.rst:146]: >one monitor at a time.  You can however use multiple DNS A records in a round-robin fashion to spread connections over multiple monitors.

File: doc/rbd/rbd-cloudstack.rst [hunk 2]
```diff
@@ -117,7 +117,7 @@ To add a Ceph block device as Primary Storage, the steps include:
    
 #. Add cluster information (``cephx`` is supported).
 
-   - For **RADOS Monitor**, provide the IP address of a Ceph monitor node.
+   - For **RADOS Monitor**, provide the IP address, comma-separated IP addresses or DNS name of a Ceph monitor nodes. Please note that the support for comma-separated multiple IP addresses of monitors are only supported from Cloudstack v4.18.0.0.
```
Comment (anthonyeleven) [inline doc/rbd/rbd-cloudstack.rst:120]: >addresses, or DNS name of Ceph monitors. Please note that support for comma-separated IP addresses is only present in Cloudstack v4.18.0.0 or later.

**Review by anthonyeleven — CHANGES_REQUESTED** [review]

**Review by zap51 — COMMENTED** [review]
Added requested changes.

File: doc/rbd/rbd-cloudstack.rst (hunk 1 above)
Comment (zap51) [reply doc/rbd/rbd-cloudstack.rst]: @anthonyeleven Requested changes have been made.

File: doc/rbd/rbd-cloudstack.rst (hunk 2 above)
Comment (zap51) [reply doc/rbd/rbd-cloudstack.rst]: @anthonyeleven Requested changes have been made.

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
Thanks for contributing.  Please adjust the PR and commit descriptions according to https://docs.ceph.com/en/quincy/start/documenting-ceph/#commit-the-change .  The automated checks won't let me merge until they're compliant.

**Comment by zap51** [comment]
> Thanks for contributing. Please adjust the PR and commit descriptions according to https://docs.ceph.com/en/quincy/start/documenting-ceph/#commit-the-change . The automated checks won't let me merge until they're compliant.

@anthonyeleven On it. Updating it now.

**Comment by zap51** [comment]
> > Thanks for contributing. Please adjust the PR and commit descriptions according to https://docs.ceph.com/en/quincy/start/documenting-ceph/#commit-the-change . The automated checks won't let me merge until they're compliant.
> 
> @anthonyeleven On it. Updating it now.

@anthonyeleven Looks like all the 3 commits are failing checks. Do I have other option than amend to make this work?

**Comment by zdover23** [comment]
> > > Thanks for contributing. Please adjust the PR and commit descriptions according to https://docs.ceph.com/en/quincy/start/documenting-ceph/#commit-the-change . The automated checks won't let me merge until they're compliant.
> > 
> > 
> > @anthonyeleven On it. Updating it now.
> 
> @anthonyeleven Looks like all the 3 commits are failing checks. Do I have other option than amend to make this work?

@zap51: 

Follow the instructions of Step 2 of this procedure: https://docs.ceph.com/en/quincy/start/documenting-ceph/#get-the-source

Squash the three commits associated with this Pull Request into a single commit, as described here: https://docs.ceph.com/en/quincy/start/documenting-ceph/#squash-extraneous-commits

Make certain that the commit message associated with the single commit that results from the squashing procedure ends with a "Signed-off-by:" line. Here is an example of a "Signed-off-by" line: Signed-off-by: Zac Dover <zac.dover@proton.me>

The only test that is failing currently is the Signed-off-by test. This can be remedied by following these instructions. When this is done, I will backport the commit associated with this PR to Reef and to Quincy.

**Review by weizhouapache — APPROVED** [review]
lgtm

**Comment by zap51** [comment]
> https://docs.ceph.com/en/quincy/start/documenting-ceph/#get-the-source



> > > > Thanks for contributing. Please adjust the PR and commit descriptions according to https://docs.ceph.com/en/quincy/start/documenting-ceph/#commit-the-change . The automated checks won't let me merge until they're compliant.
> > > 
> > > 
> > > @anthonyeleven On it. Updating it now.
> > 
> > 
> > @anthonyeleven Looks like all the 3 commits are failing checks. Do I have other option than amend to make this work?
> 
> @zap51:
> 
> Follow the instructions of Step 2 of this procedure: https://docs.ceph.com/en/quincy/start/documenting-ceph/#get-the-source
> 
> Squash the three commits associated with this Pull Request into a single commit, as described here: https://docs.ceph.com/en/quincy/start/documenting-ceph/#squash-extraneous-commits
> 
> Make certain that the commit message associated with the single commit that results from the squashing procedure ends with a "Signed-off-by:" line. Here is an example of a "Signed-off-by" line: Signed-off-by: Zac Dover [zac.dover@proton.me](mailto:zac.dover@proton.me)
> 
> The only test that is failing currently is the Signed-off-by test. This can be remedied by following these instructions. When this is done, I will backport the commit associated with this PR to Reef and to Quincy.
> 
> If any of this is too cryptic or difficult, let me know and I will step you through the procedure.

@zdover23 Really appreciate your help. I'll perform this.

File: doc/rbd/rbd-cloudstack.rst [hunk 3]
```diff
@@ -143,7 +143,7 @@ created (e.g., ``cloudstack``).
 Limitations
 ===========
 
-- CloudStack will only bind to one monitor (You can however create a Round Robin DNS record over multiple monitors)
+- CloudStack, until v4.17.1.0 will only bind to one monitor at a time. You can however use multiple DNS A or AAAA records in a round-robin fashion to spread connections over multiple monitors.
```
Comment (idryomov) [inline doc/rbd/rbd-cloudstack.rst:146]:
```suggestion
- Until v4.17.1.0, CloudStack will only bind to one monitor at a time. You can however use multiple DNS A or AAAA records in a round-robin fashion to spread connections over multiple monitors.
```

**Review by idryomov — COMMENTED** [review]
> @zdover23 Really appreciate your help. I'll perform this.

Still not quite right.  I see five commits (some of which touch the actual code and not signed).  There should be only one commit.

**Comment by zap51** [comment]
Really sorry, I'll create a new PR with the requested commit descriptions in one go. I'll also get the Git learning going on the other side.

**Pull request closed without merging**